spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/source/SparkTable.java

```diff
@@ -260,11 +260,6 @@ public MetadataColumn[] metadataColumns() {
 
   @Override
   public ScanBuilder newScanBuilder(CaseInsensitiveStringMap options) {
-    if (options.containsKey(SparkReadOptions.FILE_SCAN_TASK_SET_ID)) {
```

Comment:
We should either keep this or wait until 1.2 is out and then remove all classes and deprecated vars.

Reply:
Restored this usage.